webapp/channels/src/components/more_direct_channels/list/list.tsx

```diff
@@ -122,21 +145,7 @@ const List = React.forwardRef((props: Props, ref?: React.Ref<MultiSelect<OptionV
             noteText={note}
             maxValues={MAX_SELECTABLE_VALUES}
             numRemainingText={
-                <FormattedMessage
-                    id='multiselect.numPeopleRemaining'
-                    defaultMessage={'Use ↑↓ to browse, ↵ to select. You can\'t add more than 7 people. Please <a>create a channel</a> to include more people. '}
-                    values={{
-                        num: MAX_SELECTABLE_VALUES - props.values.length,
-                        a: (chunks: React.ReactNode) => {
-                            return (
-                                <a onClick={handleCreateChannel}>
-                                    {chunks}
-                                </a>
-                            );
-                        },
-                    }}
-                />
-
+                remaingText
```

Comment:
Is this working? The way this is right now, you are passing the function, and I would expect you to call the function and show the result instead.

Maybe it works because it considers it a function component? 0/5. Can you verify?

Reply:
I did it.

Reply:
@larkox  can you review it